.drone.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Please let's add multiple steps to the drone.yml, than it can be done in parallel and you can directly see which arch possibly failed.

Reply:
Added `.drone.yml` . See [here](https://github.com/drone-plugins/drone-hugo/pull/5/commits/d8e10b7fd3f6997b5e72e83563ec800c56e40e65).